pkg/docker/parseimage.go

```diff
@@ -26,23 +26,17 @@ import (
 	"regexp"
 )
 
-var prefix = regexp.MustCompile("docker-pullable://")
-var digestRegexp = regexp.MustCompile("@sha256:([a-zA-Z0-9]+)$")
+var digestRegexp = regexp.MustCompile("^docker-pullable://(.+)@sha256:([a-zA-Z0-9]+)$")
```

Comment:
Will this work for all openshift images?  openshift stores some images in etcd and those won't have a docker-pullable prefix.

Reply:
Fixed the issue. Please review the changes again